.github/scripts/trymerge.py

```diff
@@ -977,14 +987,21 @@ def get_diff_revision(self) -> Optional[str]:
         return rc.group(1) if rc is not None else None
 
     def has_internal_changes(self) -> bool:
-        checkrun_name = "Meta Internal-Only Changes Check"
+        checkrun_name = INTERNAL_CHANGES_CHECKRUN_NAME
         if self.get_diff_revision() is None:
             return False
         checks = self.get_checkrun_conclusions()
         if checks is None or checkrun_name not in checks:
             return False
         return checks[checkrun_name].status != "SUCCESS"
 
+    def has_no_connected_diff(self) -> bool:
```

Comment:
Returning a negative of a boolean value reduces readability, IMO replace this with `has_connected_diff`

Reply:
Sure, but is it safe to say, that `has_connect_diff()` is true if status is anything but "There is no internal Diff connected, this can be merged now"?